src/style.css

```diff
@@ -1,3 +1,115 @@
+/* Configuração geral da fonte */
 body {
-  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+/* Estilização do cabeçalho */
+header {
+  display: flex;
+  align-items: center;
+  width: 1024px;
```

Comment:
Avoid fixing the container size unless it's a specific requirement. Let the content size dictate the container size to avoid overflow or accidental scroll bars.